example/static_images.js

```diff
@@ -88,14 +88,23 @@ $(function() {
                 length: 10
             },
             decoder : {
-                readers : ["code_128_reader"]
+                readers : ["code_128_reader"],
+                drawScanline: true,
+            },
+            numOfWorkers: 4,
+            locator: {
+                halfSample: false,
+                showSkeleton: true,
+                showFoundPatches: true
             }
         }
     };
-    
+
     App.init();
+    window.App = App;
 
     Quagga.onProcessed(function(result) {
+        console.log('Quagga.onProcessed', result);
```

Comment:
removed, I just wanted to see that *something* was happening when I didn't see anything here.